Proiect-JS-main/src/index.js

```diff
@@ -0,0 +1,17 @@
+let ratioWeight = document.getElementById("ratio-weight");
+let ratioLength = document.getElementById("ratio-length");
+let weight = document.getElementById("weight");
+let length = document.getElementById("length");
+
+let calculateWeight = () => {
+  weight.value = parseFloat((weight.value * 2.2).toFixed(2));
```

Comment:
Încearcă să testezi mai bine. Eu m-am jucat puțin punând valori la întâmplare și s-a stricat.

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/2040370/227737044-a2c0e227-5bda-4ed6-b559-e7dfd4d9604f.png">